lab-abigail/index.js

```diff
@@ -0,0 +1 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

You need to require in your functions from your lib folder in here so that a user can run ``node index.js`` in their command line to run this module.